src/elements/content-uploader/UploadsManager.scss

```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
```

Comment:
So you're going to want to scope these changes for deja blue specifically and shouldn't affect non-deja blue webapp. 

So something like this is what you're going to need to do at the end of this file: 

```
.Body.is-deja-blue {
  .be.bcu-uploads-manager-container  {
 // add new styles here
}
}
```